src/canopy_server/app.py

```diff
@@ -83,7 +85,7 @@
 
 
 @app.post(
-    "/context/chat/completions",
+    f"/api/{API_VERSION}/chat/completions",
```

Comment:
Personally I find this `/api` redundant.  
Up to you, but I vote against it.

Reply:
removed